README.adoc

```diff
@@ -81,20 +83,24 @@ include::finish/ping/src/main/java/io/openliberty/guides/ping/PingHealth.java[ta
 
 This health check verifies that the `name` microservice is available at `\http://name-service:9080/api`. The `name-service` hostname is only accessible from inside the cluster, you will not be able to navigate to it yourself. If it is available then it returns an `UP` status. Similarly, if it is unavailable then it returns a `DOWN` status. When the status is `DOWN` the microservice is considered to be unhealthy.
 
+The health check for the `name` microservice has already been implemented. It has been setup to provide a sort of kill switch such that the service will be made unhealthy when a specific endpoint is called. This switch has been provided for you to observe the results of an unhealthy pod and how {kube} reacts.
```

Comment:
Not sure about the words "kill switch" here, see if you can find another wording.

Reply:
Fixed @evelinec